Makefile

```
@@ -0,0 +1,46 @@
+# Amelia development helpers
+# Postgres runs on host port 5434 (see docker-compose.yml)
+
+DB_PORT := $(or $(AMELIA_DB_PORT),5434)
+DATABASE_URL := postgresql://amelia:amelia@localhost:$(DB_PORT)/amelia_test
+
+.PHONY: db db-stop test test-unit test-integration test-all lint type-check check dev
```

checkmake (0.2.2)

[warning] Missing required phony target "all"

(minphony)